test_main.py

```diff
@@ -10,6 +10,10 @@
 import requests
 import kcidb
 
+# Disable long line checking for JSON data
+# flake8: noqa
+# pylint: disable=line-too-long
+
```

Comment:
Could you remove this particular change, and just wrap the long lines in the I/O data, now that it's not so much work?

Reply:
Okay